components/accessRequestForm/index.test.jsx

```diff
@@ -224,19 +224,19 @@ describe("Consent Request Form", () => {
   it("displays the confirmation dialog with the correct title and content regardless of the state of the toggles when clicking 'Deny All Access' button", async () => {
     const { getByTestId, getAllByTestId } = renderWithTheme(
       <ConfirmationDialogProvider>
-        <ConsentRequestContextProvider>
-          <ConsentRequestForm
+        <AccessRequestContextProvider>
+          <AccessRequestForm
             agentDetails={agentDetails}
             agentWebId={agentWebId}
           />
-        </ConsentRequestContextProvider>
+        </AccessRequestContextProvider>
       </ConfirmationDialogProvider>
     );
     const purpose = getAllByTestId(TESTCAFE_ID_PURPOSE_CHECKBOX_INPUT)[0];
     userEvent.click(purpose);
-    const toggle = getAllByTestId(TESTCAFE_ID_CONSENT_ACCESS_SWITCH)[0];
+    const toggle = getAllByTestId(TESTCAFE_ID_ACCESS_ACCESS_SWITCH)[0];
```

Comment:
this looks weird now.. maybe just GRANT_ACCESS_SWITCH? same for the string value.